spec/Section 3 -- Type System.md

```diff
@@ -1650,6 +1664,10 @@ GraphQL implementations that support the type system definition language must
 provide the `@deprecated` directive if representing deprecated portions of
 the schema.
 
+GraphQL implementations that support the type system definition language must
+provide the `@oneField` directive if representing input types that should
```

Comment:
Rather `must` instead of `should`?

I think the possibility of optional input is handled via making the input object itself null-able, but not its variants if it is a tagged union.

Reply:
How about referencing the title?

> GraphQL implementations that support the type system definition language must
provide the `@oneField` directive if representing input types that implement
the "Input Objects Requiring Exactly One Field" feature.

(Can I link to the title directly somehow?)

Reply:
Title in which context? Like [this](https://github.com/benjie/graphql-spec/blob/one-field/spec/Section%205%20--%20Validation.md#input-object-with-requires-exactly-one-field)?

Reply:
Done 👍